idiss-csharp/IdissLib/Class1.cs

```diff
@@ -0,0 +1,333 @@
+﻿using System;
```

Comment:
The class file should have a less generic name than `Class1.cs`.

Reply:
Also, I think it would be an improvement to split the contents of this class into 3 files:
1. All the type definitions.
2. The JsonConverters
3. The class `public static class Idiss`